test/Microsoft.TestPlatform.Extensions.TrxLogger.UnitTests/XmlPersistenceTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.TrxLogger.UnitTests
+{
+    using Microsoft.TestPlatform.Extensions.TrxLogger.XML;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System.IO;
+
+    [TestClass]
+    public class XmlPersistenceTests
```

Comment:
Add test case to ensure string without invalid chars are not impacted because of this change.